framework/src/st_things/things_stack/things_stack.c

```diff
@@ -850,7 +847,6 @@ static void *__attribute__((optimize("O0"))) t_things_abort_loop(s_abort_s *cont
 	if (contents) {
```

Comment:
i think it's better 
`GOTO_OUT:" move to line 851

Reply:
This code is removed in https://github.com/Samsung/TizenRT/pull/2535